<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AwAverageDialogUi</class>
 <widget class="QDialog" name="AwAverageDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Averaging conditions</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/AnyWave.qrc">
    <normaloff>:/images/AnyWave_icon.png</normaloff>:/images/AnyWave_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Select conditions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Baseline correction</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkDontUse">
          <property name="text">
           <string>Do not use baseline correction</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Method:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboMethod"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Latency range:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMaxRange">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="prefix">
           <string>-</string>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMinRange">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string>ms</string>
          </property>
          <property name="prefix">
           <string>-</string>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Data filtering</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBoxRawData">
          <property name="text">
           <string>Use raw data</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="AwFilterTableView" name="filterTableView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkOfflineFiltering">
          <property name="toolTip">
           <string>Preload the whole data in memory and apply filtering (takes more time and memory)</string>
          </property>
          <property name="text">
           <string>Offline filtering</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AwFilterTableView</class>
   <extends>QTableView</extends>
   <header location="global">filter/AwFilterTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/AnyWave.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonRun</sender>
   <signal>clicked()</signal>
   <receiver>AwAverageDialogUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkDontUse</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboMethod</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkDontUse</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinMaxRange</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkDontUse</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinMinRange</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRawData</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterTableView</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRawData</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkOfflineFiltering</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
